dojo/product_type/signals.py

```diff
@@ -0,0 +1,39 @@
+from auditlog.models import LogEntry
+from django.conf import settings
+from django.contrib.contenttypes.models import ContentType
+from django.dispatch import receiver
+from django.db.models.signals import post_save, post_delete
+from django.utils.translation import gettext as _
+from django.urls import reverse
+
+from dojo.models import Product_Type
+from dojo.notifications.helper import create_notification
+
+
+@receiver(post_save, sender=Product_Type)
+def product_type_post_save(sender, instance, created, **kwargs):
+    if created:
+        create_notification(event='product_type_added',
+                            title=instance.name,
+                            product_type=instance,
+                            url=reverse('view_product_type', args=(instance.id,)))
+
+
+@receiver(post_delete, sender=Product_Type)
+def product_type_post_delete(sender, instance, **kwargs):
```

Comment:
No notification is generated if a Product Type is deleted via either UI or API. You can see here that I've created the same `test1` Product Type multiple times without any notifications for deletions

![Screenshot 2024-04-09 at 20 37 30](https://github.com/DefectDojo/django-DefectDojo/assets/1749665/ee8d2795-972d-4301-a0f7-12f0a4433a9f)